index.html

```diff
@@ -258,6 +262,60 @@ <h3>Video Content Hints</h3>
       </p></td></tr>
     </tbody></table>
   </section>
+  <section>
+    <h2>Behaviors of other components based on content-hint</h2>
+    <section>
+      <h3>Behavior of a MediaStreamTrack</h3>
+      <p>A MediaStreamTrack for which a contentHint attribute has been
+      set MUST set the following default settings for constrainable
+        values:</p>
+        <ul>
+          <li>For an audio track with the value "music",
+            echoCancellation, autoGainControl and noiseSuppression
+            default to "false".</li>
+          <li>For an audio track with the value "speech",
+            echoCancellation and autoGainControl default to
+            "true".</li>
+        </ul>
+        <p>There is no change to the MediaTrakcCapabilities based on
+          this value; constraints may still cause these settings to
+          have any legal value.
+        </p>
+    </section>
+    <section>
+      <h3>Behavior of an RTCPeerConnection</h3>
+      <p>An RTCRtpSender transmitting a MediaStreamTrack for which a
+        contentHint attribute has been
+        set MUST perform the following actions:
+      </p>
+      <ul>
+        <li>For a video track with the attribute value "motion", if
+        RTCDegradationPreference is set to "balanced" (the default),
```

Comment:
How do I get `"balanced"` regardless of what content hints say? Should we perhaps default to `"default"` instead, to let sink configurators override things, like they can with audio constraints?

Reply:
I wondered about that too, whether the default should be "", indicating "user has expressed no preference at this API, let the platform take other things into consideration".

Reply:
I took out the default and rewrote. See if this looks righter.